eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -30,7 +35,7 @@ extends:
     Use1ESOfficial: false
     stages:
     - stage: Build
-      displayName: 'SDK Generation'
+      displayName: 'SDK Generation Preview'
```

Comment:
Didn't we want to call this `[Ingore testing]` as a prefix?

Reply:
I will add [TEST IGNORE] to `spec-gen-sdk - pull request` pipeline name. But, for job as it's used for release scenario I would rather add 'preview' instead of [TEST IGNORE].